py/torch_tensorrt/dynamo/conversion/conversion.py

```diff
@@ -1,12 +1,11 @@
 from typing import Sequence, Union
 import torch
 import io
-from torch_tensorrt.fx.trt_module import TRTModule
-from torch_tensorrt.dynamo.backend._settings import CompilationSettings
-from torch_tensorrt.dynamo.fx_ts_compat.fx2trt import (
-    InputTensorSpec,
-    TRTInterpreter,
-)
+from torch_tensorrt.dynamo.runtime import TRTModule
+from torch_tensorrt.dynamo import CompilationSettings
+from torch_tensorrt import Input
```

Comment:
This change breaks some functionality in `torch.compile`. Specifically, the `Input.from_tensors(inputs)` function is different from the `InputTensorSpec` variant in that it applies checks for aspects like contiguous memory, for which some of the `FakeTensors` Dynamo provides to us at compile-time fail the check.

Reply:
`Input` could use an argument such as `disable_memory_format_check` to make it more like `InputTensorSpec`. Additionally, a method similar to `InputTensorSpec.create_inputs_from_specs` would be helpful for the converters featured in #2070.

Reply:
@gs-olive can you link to the code where this check occurs and where it causes failures in your code?

Reply:
This is where the code is referenced:
https://github.com/pytorch/TensorRT/blob/d4c9c06213602c6e8d2ce568154ed287ee67dafd/py/torch_tensorrt/dynamo/backend/conversion.py#L38-L43
This is where the failure occurs:
https://github.com/pytorch/TensorRT/blob/d4c9c06213602c6e8d2ce568154ed287ee67dafd/py/torch_tensorrt/_Input.py#L315-L323
After looking into the issue (which appears on Resnet18, but potentially other models as well), it seems that some of the intermediate `FakeTensors` do not have `is_contiguous` with either of those memory formats (though, as `FakeTensors`, they store no values). When I disable the check, there is no such issue at runtime.

Reply:
`disable_memory_format_check ` argument to `from_tensors` seems ok to me. 
`.example_tensor()` seems to be the equivalent to `create_inputs_from_specs` call. Would that be sufficient ?

Reply:
Added `disable_memory_format_check` in newest commit + the `.example_tensor` would likely work here